src/_includes/head.hbs

```diff
@@ -0,0 +1,39 @@
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<title>{{htmlWebpackPlugin.options.site.title}}</title>
+<meta name="description" content="{{htmlWebpackPlugin.options.site.description}}">
+
+<link rel="canonical" href={{[canonicalGenerator] htmlWebpackPlugin.options.site.baseUrl htmlWebpackPlugin.options.pageName}}>
+
+<link rel="icon" href={{htmlWebpackPlugin.options.site.baseUrl}}{{htmlWebpackPlugin.options.site.faviconUrl}}>
+
```

Comment:
Looks like the two `link`s above this are missing quotation marks around the URLs